app/views/admin/plugins.html.erb

```diff
@@ -29,19 +29,42 @@ See doc/COPYRIGHT.rdoc for more details.
 <% html_title l(:label_administration), l(:label_plugins) %>
 <%= toolbar title: l(:label_plugins) %>
 <% if @plugins.any? %>
-  <table class="list plugins">
-    <% @plugins.each do |plugin| %>
-      <tr class="<%= cycle('odd', 'even') %>">
-        <td><span class="name"><%=h plugin.name %></span>
-          <%= content_tag('span', h(plugin.description), :class => 'description') unless plugin.description.blank? %>
-          <%= content_tag('span', link_to(h(plugin.url), plugin.url), :class => 'url') unless plugin.url.blank? %>
-        </td>
-        <td class="author"><%= plugin.author_url.blank? ? h(plugin.author) : link_to(h(plugin.author), plugin.author_url) %></td>
-        <td class="version"><%=h plugin.version %></td>
-        <td class="configure"><%= link_to(l(:button_configure), :controller => '/settings', :action => 'plugin', :id => plugin.id) if plugin.configurable? %></td>
-      </tr>
-    <% end %>
-  </table>
+  <div class="generic-table--container">
+    <div class="generic-table--results-container">
+      <table interactive-table role="grid" class="generic-table">
+        <colgroup>
+          <col>
+          <col>
+          <col>
+          <col>
+        </colgroup>
+        <tbody>
+          <% @plugins.each do |plugin| %>
+            <tr>
+              <td><span class="name"><%=h plugin.name %></span>
+                <%= content_tag('span', h(plugin.description), :class => 'description') unless plugin.description.blank? %>
+                <%= content_tag('span', link_to(h(plugin.url), plugin.url), :class => 'url') unless plugin.url.blank? %>
+              </td>
+              <td class="author"><%= plugin.author_url.blank? ? h(plugin.author) : link_to(h(plugin.author), plugin.author_url) %></td>
+              <td class="version"><%=h plugin.version %></td>
+              <td class="configure"><%= link_to(l(:button_configure), :controller => '/settings', :action => 'plugin', :id => plugin.id) if plugin.configurable? %></td>
+            </tr>
+          <% end %>
+        </tbody>
+      </table>
+      <div class="generic-table--header-background"></div>
+    </div>
+  </div>
 <% else %>
-  <p class="nodata"><%= l(:label_no_data) %></p>
+  <div class="generic-table--container">
+    <div class="generic-table--no-results-container">
+      <h2 class="generic-table--no-results-title">
+        <i class="icon-info"></i>
+        Nothing to display
```

Comment:
Those strings need to be internationalized (please also see the other tables). That means:
* adding a key to `config/en.yml` 
* using the key here: <%= l(:[Key]) %>